cli/pkg/timescaledb_secrets/create_secrets.go

```diff
@@ -0,0 +1,109 @@
+package timescaledb_secrets
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/timescale/tobs/cli/pkg/k8s"
+)
+
+func CreateTimescaleDBSecrets(releaseName, namespace string, enableS3Backup bool) error {
+	// Previous helm install used to create namespace if it doesn't exist
+	// but as we are creating secrets prior to deploying tobs. We are verifying
+	// namespace if doesn't create one.
+	err := k8s.CreateNamespaceIfNotExists(namespace)
+	if err != nil {
+		return err
+	}
+
+	err = k8s.CreateTimescaleDBCertificates(releaseName, namespace)
+	if err != nil {
+		return err
+	}
+
+	err = k8s.CreateTimescaleDBCredentials(releaseName, namespace)
+	if err != nil {
+		return err
+	}
+
+	if enableS3Backup {
+		err = createS3BackupForTimescaleDB(releaseName, namespace)
+		if err != nil {
+			return err
+		}
+	} else {
+		fmt.Println("skipping to create TimescaleDB s3 backup secret as backup option is disabled.")
+	}
+
+	return nil
+}
+
+func createS3BackupForTimescaleDB(releaseName, namespace string) error {
+	s3 := k8s.S3Details{}
+	//fmt.Print("We'll be asking a few questions about S3 buckets, keys, secrets and endpoints.\n\nFor background information, visit these pages:\n\nAmazon Web Services:\n- https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html\n- https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html\n\nDigital Ocean:\n- https://developers.digitalocean.com/documentation/spaces/#aws-s3-compatibility\n\nGoogle Cloud:\n- https://cloud.google.com/storage/docs/migrating#migration-simple\n\n")
+	fmt.Print(`
+We'll be asking a few questions about S3 buckets, keys, secrets and endpoints.
+
+For background information, visit these pages:
+
+Amazon Web Services:
+- https://docs.aws.amazon.com/AmazonS3/latest/gsg/CreatingABucket.html
+- https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html
+
+Digital Ocean:
+- https://developers.digitalocean.com/documentation/spaces/#aws-s3-compatibility
+
+Google Cloud:
+- https://cloud.google.com/storage/docs/migrating#migration-simple
+
+`)
+	// if values are available in env variables pick up them else
+	// read from the user input.
+	s3.BucketName = os.Getenv("PGBACKREST_REPO1_S3_BUCKET")
```

Comment:
Are these environmental variable names documented anywhere by us? What do you think of documenting them either in the prompt text or in the values.yml (next to the backup enabled section)?

Reply:
Added the docs in `values.yaml` just above the `backup` option.